src/pyenphase/auth.py

```diff
@@ -130,6 +137,20 @@ def cookies(self) -> dict[str, str]:
     def is_consumer(self) -> bool:
         return self._is_consumer
 
+    @property
+    def auth(self) -> None:
+        """No auth required for token authentication."""
+        return None
+
+    @property
+    def headers(self) -> dict[str, str]:
+        """Return the headers for Envoy authentication."""
+        return {"Authorization": f"Bearer {self.token}"}
+
+    def get_endpoint_url(self, endpoint: str) -> str:
+        """Return the URL for the endpoint."""
+        return f"https://{self.host}{endpoint}"
```

Comment:
This one is https, whilst the legacy is http. Is that correct? Just to double check

Reply:
yes